I'm unable to apply the code signatures. E.g. for arm64:

```
HOSTS="arm64-apple-darwin" ./contrib/guix/guix-codesign 
Checking that we can connect to the guix-daemon...

Hint: If this hangs, you may want to try turning your guix-daemon off and on
      again.

INFO: Codesigning 096525e92cc2 for platform triple arm64-apple-darwin:
      ...using reference timestamp: 1733177891
      ...from worktree directory: '/home/sjors/bitcoin'
          ...bind-mounted in container to: '/bitcoin'
      ...in build directory: '/home/sjors/bitcoin/guix-build-096525e92cc2/distsrc-096525e92cc2-arm64-apple-darwin-codesigned'
          ...bind-mounted in container to: '/distsrc-base/distsrc-096525e92cc2-arm64-apple-darwin-codesigned'
      ...outputting in: '/home/sjors/bitcoin/guix-build-096525e92cc2/output/arm64-apple-darwin-codesigned'
          ...bind-mounted in container to: '/outdir-base/arm64-apple-darwin-codesigned'
      ...using detached signatures in: '/home/sjors/bitcoin-detached-sigs'
          ...bind-mounted in container to: '/detached-sigs'
Required environment variables as seen inside the container:
    CODESIGNING_TARBALL: /outdir-base/arm64-apple-darwin/bitcoin-096525e92cc2-arm64-apple-darwin-codesigning.tar.gz
    DETACHED_SIGS_REPO: /detached-sigs
    DIST_ARCHIVE_BASE: /outdir-base/dist-archive
    DISTNAME: bitcoin-096525e92cc2
    HOST: arm64-apple-darwin
    SOURCE_DATE_EPOCH: 1733177891
    DISTSRC: /distsrc-base/distsrc-096525e92cc2-arm64-apple-darwin-codesigned
    OUTDIR: /outdir-base/arm64-apple-darwin-codesigned
Traceback (most recent call last):
  File "/gnu/store/jh33r2k4x0kakg4apbb2id0w8xmjxbxk-python-signapple-0.2.0-1.a9bf003/bin/.signapple-real", line 5, in <module>
    sys.exit (mod.main ())
  File "/gnu/store/jh33r2k4x0kakg4apbb2id0w8xmjxbxk-python-signapple-0.2.0-1.a9bf003/lib/python3.10/site-packages/signapple/__init__.py", line 191, in main
    args.func(args)
  File "/gnu/store/jh33r2k4x0kakg4apbb2id0w8xmjxbxk-python-signapple-0.2.0-1.a9bf003/lib/python3.10/site-packages/signapple/__init__.py", line 38, in apply
    ret = apply_sig(args.filename, args.sig)
  File "/gnu/store/jh33r2k4x0kakg4apbb2id0w8xmjxbxk-python-signapple-0.2.0-1.a9bf003/lib/python3.10/site-packages/signapple/sign.py", line 1176, in apply_sig
    shutil.copyfile(sig_file_path, file_out_path)
  File "/gnu/store/yvcl464i2vxpcq67xqhvmlkpgfbgibhb-python-3.10.7/lib/python3.10/shutil.py", line 254, in copyfile
    with open(src, 'rb') as fsrc:
FileNotFoundError: [Errno 2] No such file or directory: 'codesignatures/osx/arm64-apple-darwin/dist/Bitcoin-Qt.app/'
```

I put the mac and windows archives in `bitcoin-detached-sigs` and extractd them with `tar xf`, which looks like this:

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	osx/arm64-apple-darwin/
	osx/x86_64-apple-darwin/
	win/bitcoin-096525e92cc2-win64-setup-unsigned.exe.pem
	win/bitcoin-096525e92cc2/
```

It might be easier to have a tag or commit on that repo to checkout.

--- a/contrib/guix/libexec/codesign.sh
+++ b/contrib/guix/libexec/codesign.sh
@@ -83,7 +83,7 @@ mkdir -p "$DISTSRC"
             ;;
         *darwin*)
             # Apply detached codesignatures to dist/ (in-place)
-            signapple apply dist/Bitcoin-Qt.app codesignatures/osx/dist
+            signapple apply dist/Bitcoin-Qt.app codesignatures/osx/dist/Bitcoin-Qt.app
 
             # Make a .zip from dist/
             cd dist/

--- a/contrib/guix/manifest.scm
+++ b/contrib/guix/manifest.scm
@@ -15,13 +15,14 @@
              (gnu packages mingw)
              (gnu packages pkg-config)
              ((gnu packages python) #:select (python-minimal))
-             ((gnu packages python-build) #:select (python-tomli))
+             ((gnu packages python-build) #:select (python-tomli python-poetry-core))
              ((gnu packages python-crypto) #:select (python-asn1crypto))
              ((gnu packages tls) #:select (openssl))
              ((gnu packages version-control) #:select (git-minimal))
              (guix build-system cmake)
              (guix build-system gnu)
              (guix build-system python)
+             (guix build-system pyproject)
              (guix build-system trivial)
              (guix download)
              (guix gexp)
@@ -381,10 +382,10 @@ specific moment in time, whitelisting and revocation checks.")
       (license license:expat))))
 
 (define-public python-signapple
-  (let ((commit "62155712e7417aba07565c9780a80e452823ae6a"))
+  (let ((commit "85bfcecc33d2773bc09bc318cec0614af2c8e287"))
     (package
       (name "python-signapple")
-      (version (git-version "0.1" "1" commit))
+      (version (git-version "0.2.0" "1" commit))
       (source
        (origin
          (method git-fetch)
@@ -394,13 +395,14 @@ specific moment in time, whitelisting and revocation checks.")
          (file-name (git-file-name name commit))
          (sha256
           (base32
-           "1nm6rm4h4m7kbq729si4cm8rzild62mk4ni8xr5zja7l33fhv3gb"))))
-      (build-system python-build-system)
+           "17yqjll8nw83q6dhgqhkl7w502z5vy9sln8m6mlx0f1c10isg8yg"))))
+      (build-system pyproject-build-system)
       (propagated-inputs
         (list python-asn1crypto
               python-oscrypto
               python-certvalidator
               python-elfesteem))
+      (native-inputs (list python-poetry-core))
       ;; There are no tests, but attempting to run python setup.py test leads to
       ;; problems, just disable the test
       (arguments '(#:tests? #f))

